openquake/calculators/tests/event_based_risk_test.py

```diff
@@ -170,15 +170,6 @@ def test_case_4_hazard(self):
         fnames = export(('hmaps', 'xml'), self.calc.datastore)
         self.assertEqual(len(fnames), 4)  # 2 IMT x 2 poes
 
-        # export a single rupture
-        [f1, f2] = export(('gmfs:13', 'csv'), self.calc.datastore)
-        self.assertEqualFiles(
-            'expected/gmf-trt=05'
-            '~ses=0001~src=AS_TRAS458~rup=601989-01-PGA.csv', f1)
-        self.assertEqualFiles(
-            'expected/gmf-trt=05'
-            '~ses=0001~src=AS_TRAS458~rup=601989-01-SA(0.5).csv', f2)
-
```

Comment:
I am removing the event based GMF export tests since this is an undocumented feature that maybe remove soon or later.